### DISCUSSION TITLE
Brainstorming for sprint-2

### DISCUSSION THREAD

**Discussion by BHAVANA1497** [BHAVANA1497/GatorDesk#28] [Announcements]
Hi Team,
Let's discuss the tasks to be done in this sprint, if there are any suggestions/ideas please post them here.

**Comment by akhil-maddukuri** [comment]
We can work on the resident registration and resident login features along with things like maintaining user sessions etc., What do you think?

**Reply by Nitya-vaishnavi** [reply]
Yes @akhil-maddukuri , I believe we can work on those tasks from backend perspective. We can work on user sessions and writing code for enhancing the security. Parallely we can also work on the user sign up and registration APIs.
@BHAVANA1497 @HarshithaMyadam What do you think?

**Reply by akhil-maddukuri** [reply]
How about adding Cookies ?

**Reply by Nitya-vaishnavi** [reply]
@akhil-maddukuri Sounds great!

**Comment by BHAVANA1497** [comment]
@HarshithaMyadam Lets explore and write some test cases using cypress
